ui/src/utils/plugins.js

```diff
@@ -64,7 +64,8 @@ export const pollJobPlugin = {
         status: 'progress'
       })
 
-      options.originalPage = options.originalPage ? options.originalPage : this.$router.currentRoute.path
+      const originalPageReceive = originalPage || this.$router.currentRoute.path
```

Comment:
```suggestion
      const originalPageReceive = options.originalPage || this.$router.currentRoute.path
```

Can remove `originalPage = null` on 57
This way it will refresh only on the current / parent page